torch/__init__.py

```diff
@@ -1282,4 +1282,9 @@ def _register_device_module(device_type, module):
 import torch.fx.experimental.symbolic_shapes
 
 from torch import func as func
+
 from torch.func import vmap
+
+# Enable trace collection through dynolog
+from torch.profiler.profiler import _profile_using_dynolog
+_profile_using_dynolog(override=False)
```

Comment:
As mentioned in the previous comment, this is not necessary and not ok. This should be moved to the profiler code.
The profiler.py file is always imported on import of torch anyways, so it will do exactly the same thing.

Reply:
Sorry, I missed where the import profiler in torch/__init__.py. I'll change this.